quantities/src/main/java/com/decathlon/vitamin/compose/quantities/VitaminQuantities.kt

```diff
@@ -0,0 +1,238 @@
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminQuantities {
+
+    @Suppress("LongMethod")
+
+    @Composable
+    fun Default(
+        value: String,
+        modifier: Modifier = Modifier,
+        addEnabled: Boolean = true,
+        subtractEnabled: Boolean = true,
+        editTextEnabled: Boolean = true,
+        colors: QuantityColors = VitaminQuantitiesColors.default(),
+        keyboardOptions: KeyboardOptions = KeyboardOptions.Default,
+        onAddClicked: () -> Unit,
+        onSubtractClicked: () -> Unit,
+        onValueChange: (String?) -> Unit,
+    ) {
+        val alphaEnable = 1.0f
+
+        @Suppress("MagicNumber")
+        val alphaDisable = 0.38f
+
+        Row(
+            modifier = modifier
+                .height(48.dp)
+                .width(154.dp)
+        ) {
+
+            Button(
+                onClick = onSubtractClicked,
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .width(48.dp),
+                enabled = subtractEnabled,
+                colors = ButtonDefaults.buttonColors(
+                    backgroundColor = colors.backgroundColor,
+                    disabledBackgroundColor = colors.backgroundColor.copy(alpha = alphaDisable)
+                ),
+                shape = RoundedCornerShape(
+                    topStart = 4.dp,
+                    topEnd = 0.dp,
+                    bottomEnd = 0.dp,
+                    bottomStart = 4.dp
+                ),
+                border = BorderStroke(
+                    width = 1.dp,
+                    color = colors.borderColor.copy(alpha = if (subtractEnabled) alphaEnable else alphaDisable)
+                ),
+                elevation = ButtonDefaults.elevation(0.dp)
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_vtmn_subtract_fill),
+                    contentDescription = null,
```

Comment:
We should absolutely add a content description for the add and remove buttons. Please create a strings file in resource folders of the component and configure the english version only.